src/main/java/frc/robot/RobotMap.java

```diff
@@ -1,6 +1,20 @@
 package frc.robot;
 
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.units.Unit;
+
 public class RobotMap {
+    public static final double WHEEL_RADIUS = Units.inchesToMeters(3);
```

Comment:
```suggestion
    public static final double DRIVE_WHEEL_RADIUS = Units.inchesToMeters(3);
```
Be specific with names. There are many wheels on the robot, all in different sizes.